sync/sync.go

```diff
@@ -211,3 +224,12 @@ func marathonTaskIdsSet(marathonApps []*apps.App) map[apps.TaskID]struct{} {
 	}
 	return tasksSet
 }
+
+func findTask(id apps.TaskID, tasks []*apps.Task) bool {
```

Comment:
:+1: we could then fix it here [web/event_handler.go:176]https://github.com/allegro/marathon-consul/blob/c96bbf9b4fd5c33bc73b4b097844a43edc256cb4/web/event_handler.go#L176